src/overlays/actors/ovl_En_An/z_en_an.c

```diff
@@ -13,160 +20,3441 @@
+                case 0x28D1: // "You are looking for Kafei?"
+                    EnAn_ChangeAnim(this, play, ENAN_ANIM_SURPRISED);
+                    break;
+
+                case 0x28AF: // "Good night"
+                case 0x28C1: // "I need to hurry and cook for grandmother"
+                    EnAn_ChangeAnim(this, play, ENAN_ANIM_IDLE);
+                    break;
+
+                case 0x28BC: // "Where did you got that letter?"
+                    EnAn_ChangeAnim(this, play, ENAN_ANIM_IDLE_MORPH);
+                    break;
+
+                case 0x28C6: // "Don't talk to me or I'll drop this plate"
+                    EnAn_ChangeAnim(this, play, ENAN_ANIM_WAIT_WITH_TRAY);
+                    break;
+
+                case 0x28F5: // "The town will be crushed by the moon. Forget about the letter"
+                    EnAn_ChangeAnim(this, play, ENAN_ANIM_SIT_MORPH);
+                    break;
+
+                case 0x28F7: // "Everyone will be taking refuge by tomorrow"
+                case 0x28F8: // "We'll go to Romani Ranch to take refuge"
+                    EnAn_ChangeAnim(this, play, ENAN_ANIM_BROOM_IDLE);
+                    break;
+
+                case 0x28EB: // "I'm afraid to meet him..."
+                    if (this->animIndex != ENAN_ANIM_UMBRELLA_CRY) {
+                        this->stateFlags &= ~(ENAN_STATE_ENGAGED | ENAN_STATE_DRAW_KAFEI_MASK);
+                        this->stateFlags |= ENAN_STATE_LOST_ATTENTION;
+                        EnAn_ChangeAnim(this, play, ENAN_ANIM_UMBRELLA_CRY);
+                    }
+                    break;
+
+                case 0x28A4: // "That's good. We're totally booked right before the carnival"
+                case 0x28A9: // "You do have a reservation?"
+                case 0x28BE: // "From what postbox did you got that letter?!"
+                case 0x28C0: // "Not what I mean!"
+                case 0x295E: // "I need to know!"
+                    this->dialogueFunc = EnAn_DialogueFunc_80B556F8;
+                    this->dialogueFuncState = 0;
+                    break;
+
+                case 0x28E6: // "It's from Kafei"
+                    this->stateFlags &= ~ENAN_STATE_DRAW_BROOM;
+                    this->dialogueFunc = EnAn_DialogueFunc_80B55860;
+                    this->dialogueFuncState = 0;
+                    break;
+
+                case 0x1885: // "It's 8:30 pm and the Inn is locking the door"
+                case 0x28A0: // "Welcome to the Inn."
+                case 0x28A3: // "I'm sorry, there are no vacancies. We are always booked at the carnival"
+                case 0x28A5: // "Visit us again in the future"
+                case 0x28B1: // "I'm sorry, there are no vacancies"
+                case 0x28D2: // "I have a clue that will help find Kafei"
+                case 0x28D9: // "Please"
+                case 0x28DC: // "Thank you"
+                case 0x28DE: // "Mail it in the morning"
```

Comment:
Are these exact quotes? Could you remove them if so?

Reply:
No, they aren't exact, just similar.
I can remove them eitherway

Reply:
I think the quotations make them seem exact. If they're not, then maybe just removing the quotations should be fine